tensorflow/contrib/makefile/build_all_android.sh

```diff
@@ -32,13 +35,16 @@ if [[ -z "${NDK_ROOT}" ]]; then
     exit 1
 fi
 
-while getopts "Es:t:Tx:" opt_name; do
+ARCH=armeabi-v7a
```

Comment:
What is the exact difference between this and ANDROID_ARCH?

Reply:
They are the same, this is a short name as we are inside the `build_all_android.sh`.